### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by cyberman54** [cyberman54/ESP32-Paxcounter#806] [General]
v3.0.0 *maintenance release*

- Removed deprecated jfrog bintray OTA service
- Added new paxexpress OTA service
- Moved Wifi and BLE sniffing code to new external library libpax
- Increased privacy by evaluating universal (= random) MACs only
- Removed time and timezone libraries, now we use ezTime library
- Preserve time-of-day during sleep
- Removed beacon monitor mode (due to privacy concerns)
- Code sanitizations and small bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/cyberman54/ESP32-Paxcounter/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by ghost** [comment]


**Reply by cyberman54** [reply]
press clean before build

**Reply by ghost** [reply]
and i tried  to REMOVE IT MANUALY....No effort

**Reply by cyberman54** [reply]
it should build, regardless of the "can not remove..." message. Please post the build error here.

**Reply by spmrider** [reply]
@Umberto56 Your problem is not within the project or the build environment, but with your file system. I assume it's a Windows machine with NTFS? Please Check your disk and do a repair, then everything should be back to normal.
I've seen the effect before in very busy Directories (many, fast changing files as with compilations), mostly with File Systems that are being synced, f.e. with OneDrive.

**Comment by ghost** [comment]


**Reply by cyberman54** [reply]
@Umberto56 nothing to see... ? Please doublecheck your postings before you press send, thx.

**Reply by ghost** [reply]
Sorry...do not want to hesitate You.... BUT NOW  i have no more Interest  to build  it and so  GOOD BYE

**Reply by cyberman54** [reply]
@Umberto56 We're not on a commercial software support forum here, since you did not pay. Thus, we really need to say good bye at this point, since you ("Joined 7 days ago") will be blocked from this thread.

**Comment by ChrSchultz** [comment]
how can I get the paxexpress api-token ?

**Reply by cyberman54** [reply]
did you read the readme.md?

https://github.com/paxexpress/docs

**Reply by ChrSchultz** [reply]
yes, but in your Readme.md you didn't say that I have to create a package.  After this the upload works.
